packages/panels/resources/views/components/topbar/item.blade.php

```diff
@@ -28,6 +28,7 @@
             'flex items-center justify-center gap-x-2 rounded-lg px-3 py-2 text-sm font-semibold outline-none transition duration-75 hover:bg-gray-50 focus:bg-gray-50 dark:hover:bg-white/5 dark:focus:bg-white/5',
             'text-gray-700 dark:text-gray-200' => ! $active,
             'bg-gray-50 text-primary-600 dark:bg-white/5 dark:text-primary-400' => $active,
+            'flex items-center gap-x-1' => ! $url,
```

Comment:
There's already `flex`, `items-center` and `gap-x-2` classes on the element on line 28.